pints/_nested/_nestedEllipsoid.py

```diff
@@ -0,0 +1,313 @@
+#
+# Exponential natural evolution strategy optimizer: xNES
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+# Some code in this file was adapted from Myokit (see http://myokit.org)
+#
+import pints
```

Comment:
Please add a `from __future__ import print_function` here for python3 compatibility